src/Umbraco.Web.UI.Client/src/packages/data-type/index.ts

```diff
@@ -5,6 +5,7 @@ export * from './components/index.js';
 export * from './constants.js';
 export * from './repository/index.js';
 export * from './modals/constants.js';
+export * from './entity-actions/conditions/allow-delete/constants.js';
```

Comment:
For consistency, should this be `export * from './entity-actions/constants.js';` with intermediate `constants.ts` files defined at each level down?  Telle me if you think otherwise though, I'm just suggesting that looking at the lines above.